sphinx_automodapi/automodsumm.py

```diff
@@ -16,12 +16,18 @@
     * ``:classes-only:``
         If present, the autosummary table will only contain entries for
         classes. This cannot be used at the same time with
-        ``:functions-only:`` .
+        ``:functions-only:`` or ``:global-variables-only:``.
 
     * ``:functions-only:``
         If present, the autosummary table will only contain entries for
         functions. This cannot be used at the same time with
-        ``:classes-only:`` .
+        ``:classes-only:`` or ``:global-variables-only:``.
+
+    * ``:global-variables-only:``
```

Comment:
Hmm, I just followed the suggestion in the previous PR without thinking too much. I think you're right -- might as well call it `variables only`. I'll wait a bit in case you have other comments; then will make that change globally and squash the commits together.